webserver/decorators.py

```diff
@@ -1,9 +1,25 @@
-from functools import update_wrapper
+from functools import update_wrapper, wraps
 from datetime import timedelta
 from flask import request, current_app, make_response
+from flask_login import current_user
+from werkzeug.exceptions import Unauthorized
 from six import string_types
 
 
+def auth_required(f):
+    @wraps(f)
+    def decorated(*args, **kwargs):
+        api_key = request.args.get('key')
+        # TODO(roman): Add support for API keys when this is merged:
+        # https://github.com/metabrainz/acousticbrainz-server/pull/181
```

Comment:
Done. Would you like me to merge this PR as-is, or would you like to add this support first?